### PR TITLE
WebExtensions : browserAction popup support added to Firefox Android 57 

### PR DESCRIPTION
**Pull request by Baptistou**

### CODE REVIEW

**Comment by wbamberg** [comment]
Thanks for adding this! It looks this like also added browserAction.getPopup and setPopup: https://bugzilla.mozilla.org/show_bug.cgi?id=1370333, would you mind adding them to this PR as well?

**Comment by Baptistou** [comment]
Ok I added it to the PR also.

File: webextensions/api/browserAction.json [hunk 1]
```diff
@@ -204,7 +204,7 @@
                 "version_added": "57"
               },
               "firefox_android": {
-                "version_added": false
+                "version_added": "57"
```
Comment (wbamberg) [inline webextensions/api/browserAction.json]: I haven't tested this, but are you sure `openPopup` is supported in Android? According to http://searchfox.org/mozilla-central/source/mobile/android/components/extensions/schemas/browser_action.json#412 it is not.

**Comment by Baptistou** [comment]
Sorry, it was a mistake. I corrected it.

**Comment by wbamberg** [comment]
Looks great, thanks @Baptistou !

**Pull request merged**